wrench/script/headless.py

```diff
@@ -67,11 +88,31 @@ def set_osmesa_env(bin_path):
 
 extra_flags = os.getenv('CARGOFLAGS', None)
 extra_flags = extra_flags.split(' ') if extra_flags else []
-subprocess.check_call(['cargo', 'build'] + extra_flags + ['--release', '--verbose', '--features', 'headless'])
-set_osmesa_env('../target/release/')
+build_cmd = ['cargo', 'build'] + extra_flags + ['--verbose', '--features', 'headless']
+if optimized_build():
+    build_cmd += ['--release']
+
+objdir = ''
+if optimized_build():
+    objdir = '../target/release/'
+else:
+    objdir = '../target/debug/'
+
+subprocess.check_call(build_cmd)
+
+set_osmesa_env(objdir)
+
+dbg_cmd = []
+if use_rr():
+    dbg_cmd = ['rr', 'record']
+elif use_gdb():
+    dbg_cmd = [debugger(), '--args']
```

Comment:
Maybe something like the following would be worth it:

```suggestion
    dbg_cmd = [debugger(), '--args']
elif debugger():
    print("Unknown debugger: {}".format(debugger())
    sys.exit(1)
```